system_strings/sysmes_text.en

```diff
@@ -2244,7 +2244,7 @@ I rush over to Senpai
 -- 先生に駆け寄る。
 // Choice from 05_01_CIEL05_2_2B
 // @flowchart_title
-I rush over to Noel
+I rush over to Sensei
```

Comment:
Might keep the name, I don't think he ever refers to her as plain Sensei (Always Noel-sensei or Noel)

Reply:
He does multiple times in Ciel day 5 onwards
![image](https://github.com/Tsukihimates/Tsukihime-Translation/assets/90473809/54a0ed31-177a-425e-95b8-f1fca45d223c)

Reply:
welp never mind then